modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -177,15 +177,15 @@ curl http://LINK_FROM_DESIGN_CENTER/retrieve?key=MyKey
 In each command, replace `LINK_FROM_DESIGN_CENTER` with the cloudhub.io link in Design Center.
 
 For more information on moving to programmatic access of Object Store, see
-https://www.mulesoft.org/docs/site/3.9.0/apidocs/org/mule/api/store/ObjectStore.html[ObjectStore API docs] -- one of the strengths of OSv2 is that programmatic conventions that worked with OSv1 also work 
-with OSv2.
+https://www.mulesoft.org/docs/site/3.9.0/apidocs/org/mule/api/store/ObjectStore.html[Object Store API docs].
+Programmatic conventions that worked with OSv1 also work with OSv2.
```

Comment:
Mysti doesn't allow the use of "OSv2" or "OSv1" per JP translation. She wants these spelled out.